Packs/XMCyber/Integrations/XMCyberIntegration/XMCyberIntegration.py

```diff
@@ -665,7 +667,10 @@ def ip_command(xm: XM, args: Dict[str, Any]) -> CommandResults:
                 ip=ip,
                 dbot_score=dbot_score
             )
-            indicators_list.append(ip_standard_context)
+            command_results.append(CommandResults(
```

Comment:
let's add the specific raw data for this IOC.

Reply:
No data here, I'll adjust the result for this case.